registrations/test_tasks.py

```diff
@@ -193,39 +195,6 @@ def test_send_webhook(self, websocket):
 
         task.send_webhook(reg)
 
-        self.assertEqual(json.loads(responses.calls[-1].request.body), reg.status)
-        self.assertEqual(
-            responses.calls[-1].request.headers["Authorization"], "Bearer test-token"
-        )
-
-        websocket.assert_called_once_with(
-            "user.{}".format(user.id),
-            {"type": "registration.event", "data": reg.status},
-        )
-
-    @responses.activate
-    @mock.patch("registrations.tasks.group_send")
-    def test_send_webhook_no_url(self, websocket):
-        """
-        If no URL is specified, then the webhook should not be sent, but the
-        websocket message should still be sent.
-        """
-        user = User.objects.create_user("test", "test@example.org", "test")
-        source = Source.objects.create(
-            name="testsource", user=user, authority="hw_full"
-        )
-        reg = Registration.objects.create(
-            reg_type="jembi_momconnect", source=source, data={}, created_by=user
-        )
-
-        task.send_webhook(reg)
-        self.assertEqual(len(responses.calls), 0)
```

Comment:
I think we still need this test. This is testing that if no URL is specified, then it doesn't break things. Rather, it just doesn't make the request for the webhook